### ISSUE TITLE
Unless functionality not working as expected

### ISSUE THREAD

**Issue by fieglek** [vapid/vapid#139]
**Describe the bug**
The `Unless` function is not working as outlined in the documentation. Even using the exact [example from the documentation](https://docs.vapid.com/learn-more#conditionals) does not work. 

**To Reproduce**
1. Use a basic `unless` functionality in the HTML (this one is from the documentation).
```
     <ul>
        {{#section offices}}
          <li>
            <h5>{{name}}</h5>
            {{city}}, {{state}}
          </li>
        {{/section}}
        {{#unless offices}}
          <p>Sorry, we don't have any offices.</p>
        {{/unless}}
      </ul> 
```
2. Without anything added in the dashboard, both the main section code and the unless section code display. 

**Expected behavior**
If the `offices` section has records, then the main `section` block displays and the `unless` block does not.
If the `offices` section does not have records, then the main `section` block does not display and the `unless` block does. 

**Screenshots**
Screenshot of the rendered code above:
<img width="631" alt="Screen Shot 2019-04-23 at 9 55 39 PM" src="https://user-images.githubusercontent.com/18637059/56629335-a4182100-6612-11e9-90d2-4101cd4e5c6d.png">

Screenshot of empty dashboard section:
<img width="1071" alt="Screen Shot 2019-04-23 at 10 02 47 PM" src="https://user-images.githubusercontent.com/18637059/56629667-bcd50680-6613-11e9-9ae3-92902da9dd8d.png">

**Your Environment**
 - OS: High Sierra
 - Browser: Chrome, Safari
 - Node Version: v11.6.0
 - Vapid Version: 0.8.2 (also noticed this issue on 0.8.1)

**Comment by Hadr619** [comment]
Would there happen to be any updates on this issues? I've noticed this as well, would love a fix for this to add a message if no assets

**Comment by fieglek** [comment]
Nope. Never got a fix. Looks like no one is working on Vapid anymore.

**Comment by srobbin** [comment]
@fieglek Vapid is still being worked on, just at a slower pace this time of year.

Would either of you be interested in helping to troubleshoot and submit a PR?

**Comment by fieglek** [comment]
Oh awesome, thought it had been abandoned, good to know you are still working on it! Thanks for that.

**Comment by JohJakob** [comment]
Until the issue is fixed, you can use this workaround:

``` handlebars
<ul>
  {{#section offices}}
    {{#if name type=text}}
      <li>
        <h5>{{name}}</h5>
        {{city}}, {{state}}
      </li>
    {{/if}}
    {{#unless name}}
      <p>Sorry, we don’t have any offices.</p>
    {{/unless}}
  {{/section}}
</ul>
```

Instead of putting the section name into the `#if` and `#unless` conditions, use a template tag of the section. However, this only works if the template tag is required to ensure that it has a value in each section record. The `#unless` condition also needs to be inside the `#section` block.

**Comment by JohJakob** [comment]
See #186

**Comment by srobbin** [comment]
Fixed in #186
